Root.tsx

```diff
@@ -27,7 +30,10 @@ export type AuthStackParamList = {
 };
 
 export type DrawerParamList = {
-  HomeStack: undefined;
+  HomeStack: {
+    screen: keyof HomeStackParamList;
+    params?: HomeStackParamList[keyof HomeStackParamList];
```

Comment:
shouldn't  this be :
```suggestion
    params?: HomeStackParamList['Home'];
```
because it's specifically the params for Home and not EVERY other screen?

Reply:
That would work for now because we are only passing parameters to the Home screen of the HomeStack. The problem is that if in the future we add another screen to the HomeStack (which we will), and we pass a parameter for that screen, it wouldn't work.

Reply:
huh, I'm guessing I read this the wrong way. Once we have more screens we'll know if I read this wrong or not :)